### ISSUE TITLE
Dialog windows such as advance multiple days, repair details not appearing

### ISSUE THREAD

**Issue by 11acav** [MegaMek/mekhq#1977]
### Environment
What version of MekHQ does your issue apply to? version 0.46.1 stable
What operating system are you using?  Windows, Mac, or Linux? Windows 10 version 1903 OS build 18362.1016
What java version are you using? Version 8 Update 261 Build 1.8.0_261-b12

Issues:
1) I am on my way to my first contract. I have not been able to advance by multiple days. The dialog window is not opening properly. Also, when I hit my maintenance cycle, I attempted to view repair details and that window is also not opening properly.

2) I have 2 admin/ logistics but only 1 seems to be making rolls for parts acquisition.

3) Apart from the first few days, my admin has not been able to source a single new part.

Attached are the log files, savegame and screenshots.

[Screenshots.zip](https://github.com/MegaMek/mekhq/files/5151731/Screenshots.zip)
[Error.zip](https://github.com/MegaMek/mekhq/files/5151735/Error.zip)
[Iron Drakes30260409.zip](https://github.com/MegaMek/mekhq/files/5151741/Iron.Drakes30260409.zip)

**Comment by Windchild292** [comment]
1) Delete the mekhq.preferences file... I've had this myself when changing how my monitors work.
2) @HammerGS can you comment on this?
3) Luck of the dice, although there are a few known issues that have been fixed since our stable release.

**Comment by 11acav** [comment]
Thanks.

1. I tried deleting the file. Still no go on the dialog boxes.
2. Restarted a new campaign and seems to work fine now.
3. I was being dumb. Forgot that I was on a mission type which didn't allow for certain type of parts purchases.

**Comment by Windchild292** [comment]
For 1... Do you have multiple monitors?

**Comment by 11acav** [comment]
@Windchild292 I think I completely wasted your time cuz I was a dumbass. I finally figured out what went wrong. When I tried to open the dialog boxes, they are completely minimised. Unlike what I am used to, I just found out like 5 mins ago when I tried accessing the boxes again, that I need to manually drag and expand the boxes. There is nothing to click to auto-expand them.

My deepest apologies for wasting your time on this non-mystery. Thank you for trying to assist.
